scripts/helpers.ts

```diff
@@ -6,6 +6,90 @@ const KetlAttestationInterface = new ethers.utils.Interface(
   KetlAttestation__factory.abi
 )
 
+const GSNInterface = new ethers.utils.Interface([
```

Comment:
Maybe we can compile it like we're doing for our contracts? We probably don't need this, but in case you can use external library - this would be great

Reply:
We're only using the one `relayCall` function, so I didn't feel like we needed to / should use the GSN ABI. It's not really an instance of one of the GSN contracts either - it's just the interface.